Sources/ProjectSpec/Dependency.swift

```diff
@@ -22,7 +23,8 @@ public struct Dependency: Equatable {
         codeSign: Bool? = nil,
         link: Bool? = nil,
         implicit: Bool = implicitDefault,
-        weakLink: Bool = weakLinkDefault
+        weakLink: Bool = weakLinkDefault,
+        platform: Platform = .all
```

Comment:
The default value is defined in top of this struct like below code

https://github.com/yonaskolb/XcodeGen/blob/24e1ca0882a8b9a2000bfd14cfd065b207d0ddc6/Sources/ProjectSpec/Dependency.swift#L5-L7

Could you help add like `platformDefault`?

Reply:
Updated in 07df57d86dfd3172a9e49793ab04a9eb76e7414d.